react/features/filmstrip/actions.web.js

```diff
@@ -176,6 +178,20 @@ export function setVerticalViewDimensions() {
                 = thumbnails?.local?.width + TILE_VERTICAL_CONTAINER_HORIZONTAL_MARGIN + SCROLL_SIZE;
             remoteVideosContainerHeight
                 = clientHeight - (disableSelfView ? 0 : thumbnails?.local?.height) - VERTICAL_FILMSTRIP_VERTICAL_MARGIN;
+
+            if (getSourceNameSignalingFeatureFlag(state)) {
+                // Account for the height of the local screen share thumbnail when calculating the height of the remote
+                // videos container.
+                const localCameraThumbnailHeight = thumbnails?.local?.height;
```

Comment:
is `thumbnails` expected to be `undefined` in a regular flow (assuming there's no bug)? seems overly defensive

Reply:
I think it might be a possibility in the regular flow, I'm seeing a undefined check for thumbnails throughout the code